test/resources/microshift-rpm.resource

```diff
@@ -102,8 +102,17 @@ Verify MicroShift RPM Install
         ${nomtime}=    Set Variable    ${EMPTY}
     END
 
+    # Checks all files from RPM packages except config files
     ${stdout}    ${stderr}    ${rc}=    SSHLibrary.Execute Command
-    ...    rpm -qa microshift\* | xargs -I {} bash -c 'echo {}; sudo rpm -V ${nomtime} {}'
+    ...    rpm -qa microshift\* | xargs -I {} bash -c 'echo {}; sudo rpm -V --noconfig ${nomtime} {}'
+    ...    sudo=True    return_rc=True    return_stdout=True    return_stderr=True
+    Log    ${stdout}
+    Log    ${stderr}
+    Should Be Equal As Integers    0    ${rc}
+
+    # Checks only config files from RPM packages excluding modification time check
+    ${stdout}    ${stderr}    ${rc}=    SSHLibrary.Execute Command
```

Comment:
I think it might be occasion to simplify. I think that `SSHLibrary.Execute Command` logs the stdout and stderr anyway, so we could just do something like
```
    Command Should Work
    ...    rpm -qa microshift\* | xargs -I {} bash -c 'echo {}; sudo rpm -V --configfiles --nomtime {}'
```

That keyword would check the rc for us, so we'd have a two simple invocations and no boilerplate.

WDYT?